### PR TITLE
fix: sum of active effective balance calculations

### PR DESCRIPTION
**Pull request by vgorkavenko**

### CODE REVIEW

File: src/utils/validator_state.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
     MAX_EFFECTIVE_BALANCE,
     ETH1_ADDRESS_WITHDRAWAL_PREFIX,
     SHARD_COMMITTEE_PERIOD,
-    FAR_FUTURE_EPOCH,
+    FAR_FUTURE_EPOCH, EFFECTIVE_BALANCE_INCREMENT,
```
Comment (F4ever) [inline src/utils/validator_state.py:7]: spli in 2 lines

File: src/services/bunker_cases/abnormal_cl_rebase.py [hunk 2]
```diff
@@ -85,7 +85,10 @@ def _calculate_lido_normal_cl_rebase(self, blockstamp: ReferenceBlockStamp) -> G
         )
 
         normal_cl_rebase = AbnormalClRebase.calculate_normal_cl_rebase(
-            self.b_conf, mean_all_effective_balance, mean_lido_effective_balance, epochs_passed_since_last_report
+            self.b_conf,
+            max(EFFECTIVE_BALANCE_INCREMENT, mean_all_effective_balance),  # to avoid division by zero
```
Comment (F4ever) [inline src/services/bunker_cases/abnormal_cl_rebase.py]: not sure it belongs here.
Zero should be handled by calculate_normal_cl_rebase correctly

Comment (vgorkavenko) [reply src/services/bunker_cases/abnormal_cl_rebase.py]: To be honest, I didn't see any difference

File: tests/utils/test_validator_state_utils.py [hunk 3]
```diff
@@ -3,32 +3,66 @@
...
-    ),
-    (
-        {'0x0': Validator('0', '1', ValidatorStatus.ACTIVE_ONGOING,
-                          ValidatorState('0x0', '', '2', False, '', '1', '100500', '')),
-         '0x1': Validator('1', '1', ValidatorStatus.EXITED_SLASHED,
-                          ValidatorState('0x1', '', '2', True, '', '1', '200', ''))},
-        2,
-    ),
-]
+from src.utils.validator_state import calculate_total_active_effective_balance, is_on_exit, get_validator_age, \
```
Comment (F4ever) [inline tests/utils/test_validator_state_utils.py:6]: use () instead of \

File: src/services/bunker_cases/abnormal_cl_rebase.py (hunk 2 above)
Comment (F4ever) [reply src/services/bunker_cases/abnormal_cl_rebase.py]: Maybe handle zero inside?

Comment (vgorkavenko) [reply src/services/bunker_cases/abnormal_cl_rebase.py]: done

File: tests/utils/test_validator_state_utils.py (hunk 3 above)
Comment (vgorkavenko) [reply tests/utils/test_validator_state_utils.py]: done

File: src/utils/validator_state.py (hunk 1 above)
Comment (vgorkavenko) [reply src/utils/validator_state.py]: fixed

**Review by F4ever — APPROVED** [review]

**Pull request merged**